src/Local_Development/Extras.php

```diff
@@ -105,6 +105,21 @@ public function page_init() {
 				]
 			);
 		}
+
+		if( in_array( $_SERVER['REMOTE_ADDR'], [ '127.0.0.1', '::1' ] ) ) {
```

Comment:
My local environment updates my hosts file, so localdev.test opens. How does this change work with something like that?